modules/apps/segments/segments-test/src/testIntegration/java/com/liferay/segments/service/test/SegmentsExperienceLocalServiceTest.java

```diff
@@ -304,59 +314,121 @@ public void testFetchSegmentsExperience() throws Exception {
 	}
 
 	@Test
-	public void testMoveSegmentsExperienceToInvalidPosition() throws Exception {
-		SegmentsExperience segmentsExperience =
+	public void testMoveSegmentsExperienceToValidPosition() throws Exception {
```

Comment:
I'm not happy with the name of moveTo... I know that It was me who came up with this name :(.

 what do you think about renaming to testUpdateSegmentsExperiencePriority?
 
 the replacement consists of:
- testMoveSegmentsExperienceToValidPosition -> testUpdateSegmentsExperiencePriority
- testMoveSegmentsExperienceToZero -> testUpdateSegmentsExperiencePriorityToDefaultSegmentsExperiencePriority
- testMoveSegmentsExperienceToZeroWithBelowSegmentsExperiences -> testUpdateSegmentsExperiencePriorityToDefaultSegmentsExperiencePriorityWithLowerSegmentsExperiences
- testMoveSegmentsExperienceUnderZeroWithAboveSegmentsExperiences -> testUpdateSegmentsExperiencePriorityToDefaultSegmentsExperiencePriorityWithHigherSegmentsExperiences

Reply:
Good idea, I will add this suggestion to the same ticket, for review all the old test.

Reply:
perfect ;)